test/e2e/run.sh

```diff
@@ -108,7 +108,7 @@ prepare() {
 
   (
     cd "$rn_ui_dir"
-    npm i -g expo-cli
+    npm i expo-cli
```

Comment:
Is this confirmed to work? I believe the expo-cli must be installed globally: https://docs.expo.dev/get-started/installation/#installing-expo-cli

Reply:
ok, if it passes CI i guess it’s ok :)